examples/README.md

```diff
@@ -0,0 +1,14 @@
+# .NET for Apache Spark Samples
+
+[.NET for Apache Spark](https://dot.net/spark) is a free, open-source, and cross-platform big data analytics framework.
+
+In the **examples** folder, we provide samples which will help you get started with .NET for Apache Spark
+and demonstrate how to infuse big data analytics into existing and new .NET apps. 
+
+There are two broader categories of .NET for Apache Spark samples:
```

Comment:
Can we change this to -> 'There are two broad categories ...'

Reply:
Done, thanks!